SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -1865,6 +1865,8 @@ void CDriver::Numerics_Preprocessing(CConfig *config, CGeometry **geometry, CSol
       else if (config->GetAxisymmetric() == YES) {
         if (incompressible)
           numerics[iMGlevel][FLOW_SOL][source_first_term] = new CSourceIncAxisymmetric_Flow(nDim, nVar_Flow, config);
+        if (config->GetKind_FluidModel() == 0)
+          numerics[iMGlevel][FLOW_SOL][source_first_term] = new CSourceGeneralAxisymmetric_Flow(nDim, nVar_Flow, config);
```

Comment:
Does the general (non ideal?) implementation only work for standard air? (I did not think our standard air was non ideal)
Don't compare to 0, use the appropriate enum values.

Reply:
Oh yes the 0 was a placeholder but a bad idea. I made it default to the general version now for anything but ideal gas and standard air. I think that is how it should be since the non-ideal models use the generalised solvers for fluxes too

Reply:
It is completely general and works with any eos such as my table interpolation. Which btw works very well but before I can propose to integrate that I want to create a user interface in the config file to provide the data in ascii files. Standard air is of course ideal gas.